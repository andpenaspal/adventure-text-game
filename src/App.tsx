import { ThemeProvider as EmotionThemeProvider } from '@emotion/react';
import { ThemeProvider } from '@mui/material';

import theme from '~/theme/theme';

const App = () => {
  return (
    <>
      <ThemeProvider theme={theme}>
        <EmotionThemeProvider theme={theme}>
          <div>Hello</div>
        </EmotionThemeProvider>
      </ThemeProvider>
    </>
  );
};

export default App;
