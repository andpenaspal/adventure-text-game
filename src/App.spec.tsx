import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';

import App from './App';

describe('asfd', () => {
  test('asfd', () => {
    expect(true).toBeTruthy();
  });

  test('asdffads', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
