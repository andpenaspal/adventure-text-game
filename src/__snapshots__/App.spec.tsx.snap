// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`asfd > asdffads 1`] = `
<div>
  <div>
    Hello
  </div>
</div>
`;
